packages/web-api-scan-runner/src/runner/runner.ts

```diff
@@ -37,25 +37,31 @@ export class Runner {
 
     public async run(): Promise<void> {
         let browser: Browser;
-        let pageScanResult: OnDemandPageScanResult;
         const scanMetadata = this.scanMetadataConfig.getConfig();
 
         const scanStartedTimestamp: number = Date.now();
         const scanSubmittedTimestamp: number = this.guidGenerator.getGuidTimestamp(scanMetadata.id).getTime();
 
         this.logger.logInfo(`Reading page scan run result.`);
-        pageScanResult = await this.onDemandPageScanRunResultProvider.readScanRun(scanMetadata.id);
-        this.logger.setCustomProperties({ scanId: scanMetadata.id, batchRequestId: pageScanResult.batchRequestId });
+        this.logger.setCustomProperties({ scanId: scanMetadata.id });
 
         this.logger.trackEvent('ScanTaskStarted', undefined, { scanWaitTime: (scanStartedTimestamp - scanSubmittedTimestamp) / 1000 });
 
         this.logger.logInfo(`Updating page scan run result state to running.`);
-        pageScanResult = this.resetPageScanResultState(pageScanResult);
-        pageScanResult = await this.onDemandPageScanRunResultProvider.updateScanRun(pageScanResult);
+        const pageScanResult: Partial<OnDemandPageScanResult> = {
+            id: scanMetadata.id,
+            run: {
```

Comment:
we may need to keep the following to reset data in a storage:
`      originPageScanResult.scanResult = null;	
        originPageScanResult.reports = null;`